### PR TITLE
APR-1659 - Management Hierarchy

### PR DESCRIPTION
**Pull request by martin-hornsby**

### CODE REVIEW

File: src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+        private static List<string> TransformColumns(this List<string> tabularDataRowColumns)
+        {
+            var columns = new List<string>();
+
+            if (tabularDataRowColumns.Count == InputHeadingTitles.Count)
+            {
+                columns = new List<string>
+                {
+                    tabularDataRowColumns[0],
```
Comment (David-B-Read) [inline src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs:54]: Can you add some constants in here so it's easy to see at a glance what the array indexes refer to?

File: src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
+﻿using SFA.DAS.RoatpAssessor.Web.ApplyTypes.Apply;
```
Comment (David-B-Read) [inline src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs:1]: Test coverage for this class?

File: src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs (hunk 1 above)
Comment (martin-hornsby) [reply src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs]: Sure, have added these

File: src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs (hunk 2 above)
Comment (martin-hornsby) [reply src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Transformers/ManagementHierarchyTransformer.cs:1]: Added 8 test cases to provide requested coverage

**Pull request merged**